src/blocks/DocActions/view.js

```
@@ -0,0 +1,208 @@
+					markdown += text + '\n';
+				}
+			} else if (node.nodeType === Node.ELEMENT_NODE) {
+				const tag = node.tagName.toLowerCase();
+				switch (tag) {
+					case 'h1':
+					case 'h2':
+					case 'h3':
+					case 'h4':
+					case 'h5':
+					case 'h6':
+						const level = parseInt(tag[1]);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)